controllers/defaults/placements.go

```diff
@@ -76,7 +79,10 @@ var (
 			},
 			PodAntiAffinity: &corev1.PodAntiAffinity{
 				PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{
-					getWeightedPodAffinityTerm(100, "rook-ceph-mds"),
+					// the selector `MdsNodeAffinitySelector` value will be changed later in the getPlacement(),
+					// to the actual value of the filesystem,
+					// do not append the list of values here as later it will just replace the first value
```

Comment:
We can append it, Pls mention we are using the 0 index to change the value. So 0 index should remain 0.